cmd/did-method-cli/sample/publickeys.json

```diff
@@ -0,0 +1,26 @@
+[
+ {
+  "id": "key1",
+  "type": "Ed25519VerificationKey2018",
+  "encoding": "Jwk",
+  "keyType": "Ed25519",
+  "purpose": ["verificationMethod"],
```

Comment:
you may want to change purpose to purposes in order to match Sidetree

Reply:
Isn't this required @sandrask ?

Reply:
done